### ISSUE TITLE
Different kinds of power plants

### ISSUE THREAD

**Issue by spirulence** [spirulence/power-department-ld38#11]
There should be some diversity in the kinds of power plants that can be created. Each should have distinct advantages inspired by real-world options.

- [ ] solar
- [ ] wind
- [ ] hydro
- [ ] coal
- [ ] natural gas (with combined cycle upgrade)
- [ ] nuclear

Any others that we should talk about?

**Comment by caryoscelus** [comment]
Good idea. But they should be presented gradually, since when the game starts power industry is just being created.

Also, if being ambitious, we may add some notion of helper facilities (e.g. coal mines). From realistic point of view they shouldn't be user-built and controlled. Perhaps they can even work similar to cities, providing coal price gradient and alike.
